token-syncer/src/token_syncer/waiter.clj

```diff
@@ -195,3 +195,26 @@
       {:body (parse-json-data body-data)
        :headers (extract-relevant-headers headers)
        :status status})))
+
+(defn health-check-token
+  "Performs health check on a token on a specific cluster."
+  [^HttpClient http-client cluster-url token]
+  (log/info "health-check-token" token "on" cluster-url)
+  (let [{:keys [description]} (load-token http-client cluster-url token)
+        {:strs [deleted health-check-url]} description]
+    (if (and (not deleted) (not (str/blank? health-check-url)))
+      (do
+        (log/info "health-check-token" token "on" (str cluster-url health-check-url))
+        (-> (make-http-request http-client (str cluster-url health-check-url)
+                               :headers {"x-waiter-queue-timeout" (-> 2 t/minutes t/in-millis)
```

Comment:
Should this be configurable?

Reply:
Done, added a `-q` flag to configure this in the sub-command.

Reply:
Is there a long-form for it as well?

Reply:
Yes, `--queue-timeout`